WebApiBackend/WebApiBackend/Migrations/FlatManagementContextModelSnapshot.cs

```diff
@@ -0,0 +1,214 @@
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("INTEGER");
+
+                    b.Property<string>("Address")
+                        .HasColumnType("TEXT");
+
+                    b.HasKey("Id");
+
+                    b.ToTable("Flat");
+                });
+
+            modelBuilder.Entity("WebApiBackend.Model.Payment", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("INTEGER");
+
+                    b.Property<int>("Amount")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<string>("Description")
+                        .HasColumnType("TEXT");
+
+                    b.Property<DateTime>("EndDate")
+                        .HasColumnType("TEXT");
+
+                    b.Property<bool>("Fixed")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<int?>("FlatId")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<string>("Frequency")
+                        .IsRequired()
+                        .HasColumnType("TEXT");
+
+                    b.Property<string>("PaymentType")
+                        .IsRequired()
+                        .HasColumnType("TEXT");
+
+                    b.Property<DateTime>("StartDate")
+                        .HasColumnType("TEXT");
+
+                    b.HasKey("Id");
+
+                    b.HasIndex("FlatId");
+
+                    b.ToTable("Payment");
+                });
+
+            modelBuilder.Entity("WebApiBackend.Model.Schedule", b =>
+                {
+                    b.Property<string>("UserName")
+                        .HasColumnType("TEXT");
+
+                    b.Property<DateTime>("StartDate")
+                        .HasColumnType("TEXT");
+
+                    b.Property<DateTime>("EndDate")
+                        .HasColumnType("TEXT");
+
+                    b.Property<string>("ScheduleType")
+                        .HasColumnType("TEXT");
+
+                    b.Property<int?>("FlatId")
+                        .HasColumnType("INTEGER");
+
+                    b.HasKey("UserName", "StartDate", "EndDate", "ScheduleType");
+
+                    b.HasIndex("FlatId");
+
+                    b.ToTable("Schedule");
+                });
+
+            modelBuilder.Entity("WebApiBackend.Model.TestModelItem", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("INTEGER");
+
+                    b.Property<string>("Name")
+                        .HasColumnType("TEXT");
+
+                    b.HasKey("Id");
+
+                    b.ToTable("TestItems");
+                });
+
+            modelBuilder.Entity("WebApiBackend.Model.User", b =>
+                {
```

Comment:
Hey Lucas, we're not using migrations at all. Could you clean these out?

The database should autopopulate on launch, and is reset everytime (in future I guess the application would need something for prod, but this is suitable for the dev phase).

Reply:
I'll run this and test it ASAP :).

Reply:
Have removed the folder. Please confirm :)